### ISSUE TITLE
loss around 20 but yolo_video.py won't detect object

### ISSUE THREAD

**Issue by YHL04** [qqwweee/keras-yolo3#656]
Loss pretty stable during stage 1 but when testing it with one of the training data images it doesn't detect the object

**Comment by YHL04** [comment]
some training data didnt have object in it so it learned to not detect object to reduce loss, took out images without training object and it worked
